src/utils/gh-auth.js

```diff
@@ -86,62 +69,29 @@ const authWithNetlify = async ({ log }) => {
   return deferredPromise
 }
 
-const getUsernameAndPassword = async () => {
-  const { username, password } = await inquirer.prompt([
+const getPersonalAccessToken = async () => {
```

Comment:
Username and password auth has been deprecated by GitHub. See https://github.com/netlify/cli/issues/2229#issuecomment-828483728

This is technically a breaking change as we're dropping support for it in favor of a personal access token, but since it's already broken I don't think it justifies a major version bump.